### PR TITLE
RX tab refactor

### LINKED ISSUE #57: not possible to save bind data for RX
Apparently, there's currently no way to save the current `BIND_DATA` structure to the RX when connecting remotely from a TX to an RX. So, an RX isn't automatically bound to the current TX when connecting remotely from TX->RX and clicking the "save to EEPROM" button.

IOW, just because you connect remotely from a TX to an RX module, it doesn't mean that they're consequently bound to each other. I think this goes against what the expected behavior might be in many cases...

I feel that, at the very least, we should have another button (labeled "Bind Receiver" or something of this nature) that allows the user to save the current `BIND_DATA` structure from the TX to the currently-connected RX, i.e., 'bind' the RX to the TX without having to go through the process of binding via some other means.

@mikeller & others, feedback needed...

### PR DESCRIPTION
**Pull request by csurf**
- complete refactor of RX tab code
- add support for binding receiver remotely via wireless connection from
TX to RX
- adds 'Bind to TX' button when connected to RX remotely. clicking button saves current bind data from TX on RX
- 'Bind to TX' button is hidden when connected directly to RX
- small changes to fix some jquery selector bug/errors for ':last'
selector

this resolves #57 and requires fix on firmware side (PR to follow)

### CODE REVIEW

**Comment by csurf** [comment]
ref. PR on firmware side openLRSng/openLRSng#196

File: js/psp.js [hunk 1]
```diff
@@ -377,33 +383,28 @@ PSP.send_message = function (code, data, callback_sent, callback_psp, timeout) {
...
-                PSP.send_message(PSP.PSP_SET_RX_SAVE_EEPROM, false, false, (callback) ? callback : undefined);
-            }
-
-            PSP.send_message(PSP.PSP_SET_RX_CONFIG, rx_data, false, save_to_eeprom);
-        }
-    } else {
-        GUI.log(chrome.i18n.getMessage('running_in_compatibility_mode'));
-    }
+    if (CONFIGURATOR.readOnly) {
+		GUI.log(chrome.i18n.getMessage('running_in_compatibility_mode'));
```
Comment (mikeller) [inline js/psp.js]: Mixing tabs and spaces (in multiple files).

Comment (csurf) [reply js/psp.js]: ok, will fix

**Comment by csurf** [comment]
@mikeller 
ok, the last commit should only have the RX refactor and I fixed the tab & line feeds. If you OK this, I can commit the diff with the new 'bind' button

**Pull request merged**